docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -110,7 +110,9 @@ var _ = Describe("Manager", Ordered, func() {
 					"pods", controllerPodName, "-o", "jsonpath={.status.phase}",
 					"-n", namespace,
 				)
-				g.Expect(utils.Run(cmd)).To(BeEquivalentTo("Running"), "Incorrect controller-manager pod status")
+				output, err := utils.Run(cmd)
+				g.Expect(err).NotTo(HaveOccurred())
+				g.Expect(string(output)).To(BeEquivalentTo("Running"), "Incorrect controller-manager pod status")
```

Comment:
To avoid using BeEquivalentTo (to keep things simple), let's just use Equal here.

```suggestion
				g.Expect(string(output)).To(Equal("Running"), "Incorrect controller-manager pod status")
```

Reply:
That is fine but I think is out of the scope of the PR right?
I think that is not related with the issue/motivation